logdna/configs.py

```diff
@@ -1,4 +1,7 @@
-from _version import __version__
+import os.path
+import sys
+import inspect
```

Comment:
I don't think you need the followings here:
```
import os.path
import sys
import inspect
```

Reply:
oh yes forgot to remove this ones